DictionaryCompressor/FileCompressor.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+
+namespace DictionaryCompressor
+{
+    internal class FileCompressor : IFileCompressor
+    {
+        public void CompressFile(string filePath, string output)
+        {
+            FileInfo file = new FileInfo(filePath);
+            using (FileStream originalFileStream = file.OpenRead())
```

Comment:
This is not unit testable, as you use the filesystem directly. Use [`Messerli.FileOpeningBuilder`](https://github.com/messerli-informatik-ag/file-opening-builder) by injection